lib/error-helper.js

```diff
@@ -1,19 +1,18 @@
 'use strict'
 
 const { NotFound, InvalidResponse } = require('../services/errors')
+const defaultErrorMessages = {
+  404: 'not found',
```

Comment:
Do you think it would be worth while adding a few more defaults?
```http
401: Unauthorized
403: Forbidden
408: Request Timeout
429: Too Many Requests
500: Internal Server Error
```
_Maybe some others too, not really sure which would be the more common ones we would possibly encounter_

Reply:
I don't believe there is, but it would likely [fail due to timeouts](https://circleci.com/gh/badges/shields/9449).
Good point, i'm fine with just having 404 default for now, until we see a need for more.